include/solvers/multiphysics_interface.h

```diff
@@ -493,6 +493,53 @@ class MultiphysicsInterface
     return physics_solutions[physics_id];
   }
 
+  /**
+   * @brief Request the average solution of a given physics
```

Comment:
Same as for the parameter, this comment could make it more clear that you are aiming to use a pseudo steady state solution, not being confused with a spatial average.